services/github/github-commit-activity.service.js

```diff
@@ -31,6 +41,7 @@ export default class GitHubCommitActivity extends GithubAuthV4Service {
       // Override the pattern to omit the deprecated interval "4w".
       pattern: ':interval(t|y|m|w)/:user/:repo',
       namedParams: { interval: 'm', user: 'eslint', repo: 'eslint' },
+      queryParams: { authorFilter: 'chris48s' },
```

Comment:
Just to make the example "make sense", can we pick a username of someone who has made one or more commits to the example repos. e.g: for eslint/eslint, `nzakas` would make more sense as an example.

Reply:
I will pick one of the users for the example, no problem

Reply:
Thanks. Could we do the same on the other example (pick an author who has contributed to the repo)

Reply:
Sure